lib/opa/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3.7'
+
+services:
+  opa:
+    image: openpolicyagent/opa:${OPA_VERSION:-latest}
+    volumes:
+      # TODO: temp solution
+      - $PWD/lib/opa/opa_service:/data
+    networks:
+      - ${DOCKER_NET}
+    ports:
+      - "${OPA_PORT}:8181"
+    deploy:
+      placement:
+        constraints:
+          - node.role == worker
+      restart_policy:
+        condition: on-failure
+        delay: 5s
+        max_attempts: 3
+        window: 120s
+      labels:
+        - traefik.enable=true
+        - traefik.port=${OPA_PORT}
+        - traefik.docker.network=${DOCKER_NET}
+        - traefik.frontend.rule=Host:opa.${CANDIG_DOMAIN}
+    logging:
+      driver: "fluentd"
+      options:
+        tag: docker.{{.Name}}
+        fluentd-async-connect: "true"
+    command: ["run", "--server", "/data/metadata-service.json", "/data/metadata-service.rego"]
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8181/health"]
+      interval: 30s
+      timeout: 20s
+      retries: 3
```

Comment:
opa docker-compose.yml looks good, but see previous comments about consolidating authN/Z services

Reply:
this should not be part of this PR